Automattic/amp-wp
# Setup wizard: summary screens

## Summary


Fixes #4703
Fixes #4709
Fixes #4710 

### Changes contained

1. Creates three versions of the onboarding wizard's summary screen, one for each mode. 
2. Updates the nav component to make it fixed to the bottom.
3. Makes the minimum required updates needed to persist a "mobile_redirect" option. #4796 covers that feature in full.
4. Various style updates, including basic improvements for mobile
5. Fixed failing e2e tests that made it into `develop` on another PR
6. Fixed some minor suggestions from #4900 
7. Added a new option to persist whether the setup wizard has run once. If it has never run, it does not pre-select any defaults.
8. Adds a "Previously selected" notice on the mode that was previously stored in the database.

### Screenshots

Reader
<img width="1052" alt="Screen Shot 2020-06-23 at 2 48 00 PM" src="https://user-images.githubusercontent.com/9020968/85452862-e9aae380-b560-11ea-95f5-fe5b0c78d182.png">

Standard
<img width="1055" alt="Screen Shot 2020-06-23 at 2 48 24 PM" src="https://user-images.githubusercontent.com/9020968/85452884-ee6f9780-b560-11ea-9952-26dac71b184d.png">

Transitional
<img width="1042" alt="Screen Shot 2020-06-23 at 2 48 38 PM" src="https://user-images.githubusercontent.com/9020968/85452915-f596a580-b560-11ea-97e7-32b1cf021216.png">

## QA

1. Open the setup wizard
2. Move through the wizard and select a mode
3. Verify that the summary screen shows content corresponding to your mode, and that the summary screen looks right (copy still WIP). Verify the theme shown matches your site's theme and that the reader theme matches your selection (if you chose reader mode).
4. Repeat for each mode
5. Verify that the mode you selected previously (and saved by going all the way through the wizard) shows the "Previously selected" notice on the mode screen the next time you run through it
6. Add `&setup-wizard-first-run=1` to tell the wizard to pretend this is your first run. In this case, all options should require you to select them


## Checklist
